introduction.md

```diff
@@ -5,3 +5,7 @@ I am the ruler of this dojo.
 This is my commit.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?